src/test/java/com/github/gilday/darkmode/DarkModeDetectorTest.java

```diff
@@ -16,6 +16,6 @@ void reads_dark_mode_setting() {
     assumeTrue(SystemInfo.isMacOSMojave);
 
     // test passes if isDarkMode completes without error
-    DarkModeDetector.isDarkMode();
+    DarkModeDetector.isMacOsDarkMode();
```

Comment:
Can you add a similar test for Windows?

Reply:
I implemented a similar test as requested. Be aware, that the test only run on the corresponding OS.